kubecost/templates/network-costs/network-costs-daemonset.yaml

```
@@ -121,9 +121,19 @@ spec:
       tolerations:
 {{ toYaml .Values.networkCosts.tolerations | indent 8 }}
     {{- end }}
-      {{- with .Values.networkCosts.affinity }}
+      {{- if .Values.networkCosts.useDefaultAffinity }}
       affinity:
-        {{- toYaml . | nindent 8 }}
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: topology.kubernetes.io/region
+                operator: Exists
+              - key: topology.kubernetes.io/zone
+                operator: Exists
+        {{- else }}
```

Small style change. Indentation to match `if`.

```suggestion
      {{- else }}
```